common/persistence/nosql/nosql_execution_store.go

```diff
@@ -471,6 +485,8 @@ func (d *nosqlExecutionStore) ConflictResolveWorkflowExecution(
 			return err
 		}
 
+		// TODO(active-active): make changes here to insert active cluster selection policy.
```

Comment:
@Shaddoll FYI You were right. Conflict resolution should also insert new policy record here. I'm leaving this for later